File: Oregon 9-1-1/Oregon 9-1-1.Android.NativeActivity/changeLog.cpp
#include "Oregon911.h"

changeLog::changeLog(unsigned int _id, timestamp _ts, unsigned char _type, double _lat, double _lon, std::string _callSum, std::string _address)
{
	id = _id;
	ts = _ts;
	type = _type;
	lat = _lat;
	lon = _lon;
	callSum = _callSum;
	address = _address;
}

changeLog::changeLog() {
	id = 0;
	type = 0;
	lat = 0.0;
	lon = 0.0;
}

changeLog::~changeLog()
{

}

unsigned int changeLog::getid()
{
	return id;
}

timestamp changeLog::getTimeStamp()
{
	return ts;
}

unsigned char changeLog::getType()
{
	return type;
}

double changeLog::getLat()
{
	return lat;
}

double changeLog::getLon()
{
	return lon;
}

std::string changeLog::getCallSum()
{
	return callSum;
}

std::string changeLog::getAddress()
{
	return address;
}

void changeLog::setid(unsigned int _id)
{
	id = _id;
}

void changeLog::setTimeStamp(const timestamp & _ts)
{
	ts = _ts;
}

void changeLog::setType(const unsigned char & _type)
{
	type = _type;
}

void changeLog::setLat(const double & _lat)
{
	lat = _lat;
}

void changeLog::setLon(const double & _lon)
{
	lon = _lon;
}

void changeLog::setCallSum(const std::string& _callsum)
{
	callSum = _callsum;
}

void changeLog::setAddress(const std::string& _address)
{
	address = _address;
}

File: Oregon9-1-1/app/src/main/java/cad/oregon911/net/changeLog.java
package cad.oregon911.net;

/**
 * Created by Brandan on 12/28/2015.
 */
public class changeLog {

    private int id;
    private timestamp ts;
    private int type;
    private double lat;
    private double lon;
    private String callSum;
    private String address;

    public changeLog() {
        this.id = 0;
        this.type = 0;
        this.lat = 0.0;
        this.lon = 0.0;
        ts = new timestamp();
    }

    public changeLog(int id, timestamp ts, int type, double lat, double lon, String callSum, String address) {
        this.id = id;
        this.ts = ts;
        this.type = type;
        this.lat = lat;
        this.lon = lon;
        this.callSum = callSum;
        this.address = address;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public timestamp getTs() {
        return ts;
    }

    public void setTs(timestamp ts) {
        this.ts = ts;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public double getLat() {
        return lat;
    }

    public void setLat(double lat) {
        this.lat = lat;
    }

    public double getLon() {
        return lon;
    }

    public void setLon(double lon) {
        this.lon = lon;
    }

    public String getCallSum() {
        return callSum;
    }

    public void setCallSum(String callSum) {
        this.callSum = callSum;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }
}
